service/src/main/java/uk/nhs/adaptors/oneoneone/cda/report/mapper/ProcedureRequestMapper.java

```diff
@@ -0,0 +1,50 @@
+package uk.nhs.adaptors.oneoneone.cda.report.mapper;
+
+import static org.hl7.fhir.dstu3.model.IdType.newRandomUuid;
+
+import org.apache.commons.lang3.StringUtils;
+import org.hl7.fhir.dstu3.model.CodeableConcept;
+import org.hl7.fhir.dstu3.model.Coding;
+import org.hl7.fhir.dstu3.model.ProcedureRequest;
+import org.springframework.stereotype.Component;
+
+import lombok.RequiredArgsConstructor;
+import uk.nhs.connect.iucds.cda.ucr.POCDMT000002UK01ClinicalDocument1;
+
+@Component
+@RequiredArgsConstructor
+public class ProcedureRequestMapper {
+    public ProcedureRequest mapProcedureRequest(POCDMT000002UK01ClinicalDocument1 clinicalDocument) {
+        ProcedureRequest procedureRequest = new ProcedureRequest();
+        if (clinicalDocument.isSetComponentOf()) {
+            if (clinicalDocument.getComponentOf().getEncompassingEncounter() != null) {
+                if (clinicalDocument.getComponentOf().getEncompassingEncounter().isSetDischargeDispositionCode()) {
+                    Coding coding = new Coding();
+
+                    if (clinicalDocument.getComponentOf().getEncompassingEncounter().getDischargeDispositionCode().isSetDisplayName()) {
+                        coding.setDisplay(clinicalDocument.getComponentOf().getEncompassingEncounter().getDischargeDispositionCode()
+                            .getDisplayName());
+                    }
+                    if (clinicalDocument.getComponentOf().getEncompassingEncounter().getDischargeDispositionCode().isSetCode()) {
+                        coding.setCode(clinicalDocument.getComponentOf().getEncompassingEncounter().getDischargeDispositionCode()
+                            .getCode());
+                    }
+                    if (clinicalDocument.getComponentOf().getEncompassingEncounter().getDischargeDispositionCode().isSetCodeSystem()) {
+                        coding.setSystem(clinicalDocument.getComponentOf().getEncompassingEncounter().getDischargeDispositionCode()
+                            .getCodeSystem());
+                    }
+                    if (!StringUtils.isBlank(coding.getCode()) || !StringUtils.isBlank(coding.getDisplay())
```

Comment:
Use `StringUtils.isNotBlank(coding.getCode())` instead of negation
Also I think there was a helper `anyNotBlank` in `StringUtils` or in `CollectionUtils`